internal/charmstore/store.go

```diff
@@ -29,6 +29,11 @@ import (
 
 var logger = loggo.GetLogger("charmstore.internal.charmstore")
 
+const (
+	// StableChannel is the channel used internally for stable charms or bundles.
+	StableChannel charm.Channel = "stable"
```

Comment:
Should we define this in charm? Or should we remove the channel concept from charm and then go ahead and define the type with both stable and development channels here? We are no longer using channels in URLs, so I'd propose to just consider channels a charm store concept. So perhaps define channels here and eventually remove from charm?